source/virtualBuffers/__init__.py

```diff
@@ -109,14 +109,30 @@ def obj(self):
 
 	@property
 	def label(self):
-		if self.itemType == "landmark":

```

Comment:
I would love to see some documentation added here. Specifically talking about what `label` is and how its expected to be used. I realise that you didn't introduce this, but since you are in the area and have taken the time to understand this code it would be beneficial if this was documented better.